npm/cypress-schematic/src/schematics/ng-add/index.ts

```diff
@@ -306,6 +307,33 @@ function modifyAngularJson (options: any): Rule {
   }
 }
 
+function addDefaultSchematic (): Rule {
+  return (tree: Tree, _: SchematicContext) => {
+    if (tree.exists('./angular.json')) {
+      const angularJsonVal = getAngularJsonValue(tree)
+      const angularSchematic = '@schematics/angular'
+      const cli = {
+        ...angularJsonVal.cli,
+        schematicCollections: ['@cypress/schematic', ...angularJsonVal?.cli?.schematicCollections ?? []],
+      }
+
+      if (cli.schematicCollections.indexOf('@schematics/angular') === -1) {
```

Comment:
`schematicCollections` is only for Angular-14 apps. Previously, the value was `defaultCollection` but it isn't an array. Should we override the default value instead for Angular 13 apps?